internal/configs/virtualserver.go

```diff
@@ -1071,14 +1071,16 @@ func (p *policiesCfg) addDosConfig(
 
 		if logConfPath, ok := dosResources.LogConfs[logConfKey]; ok {
 			logDest := generateString(dos.DosSecurityLog.DosLogDest, "stderr")
-			p.Dos.ApDosLogConf = fmt.Sprintf("%s %s", logConfPath, logDest)
+			p.Dos.ApDosLogConf = fmt.Sprintf("%s %s", logConfPath, generateDosLogDest(logDest))
 		} else {
 			res.addWarningf("Dos policy %s references an invalid or non-existing log config %s", polKey, logConfKey)
 			res.isError = true
 		}
 	}
 
-	p.Dos.ApDosAccessLogDest = dos.DosAccessLogDest
+	if p.Dos.ApDosAccessLogDest != "" {
```

Comment:
`if p.Dos.ApDosAccessLogDest != "" {`
need to be 
`if dos.DosAccessLogDest != "" {`